src/state/messages/convo/agent.ts

```diff
@@ -553,7 +553,7 @@ export class Convo {
           {
             cursor: nextCursor,
             convoId: this.convoId,
-            limit: isNative ? 40 : 60,
+            limit: isNative ? 30 : 60,
```

Comment:
This needs to be the same as `initialNumToRender` in `messagesList`. If it's larger, there will be a scroll animation on render.

I'd like to keep this below 30 if we can, especially whenever we start considering adding other content types than text (honestly we might need to decrease the number when that happens, at least on native)

Reply:
Heard chef 🫡